### ISSUE TITLE
Empty ticker in CCXT download & possible strategy.exit close issue

### ISSUE THREAD

**Issue by fitz-97** [PyneSys/pynecore#30]
This open-source project is excellent, and I’d like to thank the author for their contribution. It has to some extent met some of my other needs as well. However, during use, I seem to have unintentionally discovered a few bugs.

All the following issues occurred while running in developer source mode:

1.When fetching data from the corresponding exchange via ccxt, it appears that ticker = market_details['info']['symbol'] is empty, which causes the download to fail.

2.A similar issue occurs after resolving point 1, where in syminfo the value of ticker = symbol['ticker'] is empty.
Both of these issues are triggered by running the following command:

```bash
pyne data download ccxt --symbol "okx:ETH/USDT:USDT" --timeframe 120 --from 100
```
3.Some exchanges support 120-minute timeframe — recommend enabling it and only disabling for unsupported exchanges.

4.During strategy backtesting, I carefully compared the logs and found that the values of various indicators match exactly with their counterparts in Pine Script. I think this is truly remarkable. However, theoretically, my order information should also match the results on TradingView, but in practice, they do not.
After investigation, I roughly determined that the issue is related to strategy.exit(). Perhaps there are certain scenarios where it fails to close a position. I attempted to use both trailing stop loss/take profit and fixed-amount stop loss/take profit with strategy.exit(), but the results did not match my expectations.
Below is my Pine Script code. Once again, thank you to the Pyne development team.

```python
//@version=6
strategy('SMA TEST1', overlay = true, initial_capital = 100000, commission_type = strategy.commission.percent, commission_value = 0.1)

short_length = input.int(9, ' ')
long_length = input.int(21, ' ')

short_ma = ta.sma(close, short_length)
long_ma = ta.sma(close, long_length)

plot(short_ma, color = color.red, title = ' ')
plot(long_ma, color = color.blue, title = ' ')

long_condition = ta.crossover(short_ma, long_ma)
short_condition = ta.crossunder(short_ma, long_ma)

offsetPriceTick = 0.1 / 100 * close / syminfo.mintick
trailPoints = close * 0.01 / syminfo.mintick
lossPoints = close * 0.01 / syminfo.mintick

start = timestamp(2025, 8, 1, 0, 0, 0)

window() =>
    time >= start ? true : false

if long_condition and window()
    strategy.entry('Long', strategy.long)
    strategy.exit('TP/SL Exit', loss = lossPoints, trail_points = trailPoints, trail_offset = offsetPriceTick)

```

**Comment by fitz-97** [comment]
The problem may be about this part in `strategy.exit()`:
```python
    entry_order = position.orders.get(from_entry, None)  # type: ignore
    # Find open trade if no entry order found

    if not entry_order:
        for trade in position.open_trades:
            if trade.entry_id == from_entry:
                direction = trade.sign
                size = trade.size
                break
    else:
        direction = entry_order.sign
        size = entry_order.size
    if not direction:
        return
```

It seems the generation of the content of position.open_trades may be related to the execution result of strategy.entry().
During my debugging, I noticed that if from_entry is empty, it's rarely accurate. When it's empty, the pine script defaults to creating exit orders for all open positions until they are closed.
Currently, even when specifying from_entry, the closing price doesn't seem to match expectations. I'd appreciate some help troubleshooting this.

**Comment by fitz-97** [comment]
> The bugs are fixed. And I added the 120 (min) to the allowed options. These will be in the next release.
> 
> However the strategy problem is not a bug, just the trailing stop is not yet supported. But I've started working on it. I hope it will be ready in the next one or two days.

Thank you. I'm waiting for to the next version release

**Comment by wallneradam** [comment]
It was a bit harder then I thought, but it is now available with a lot of other fixes and optimizations.
Thanks again for reporting these.

**Comment by wallneradam** [comment]
@fitz-97 All of this are fixed in the latest, very big strategy simulator update in v6.3.3.
